modules/lib/src/main/resources/assets/js/app/ui2/list/ProjectItem.tsx

```diff
@@ -0,0 +1,90 @@
+import {ListItem, type ListItemProps} from '@enonic/ui';
+import type {ReactElement} from 'react';
+import {LegacyElement} from '@enonic/lib-admin-ui/ui2/LegacyElement';
+import {Project} from '../../settings/data/project/Project';
+import {ProjectHelper} from '../../settings/data/project/ProjectHelper';
+import ProjectIcon from './ProjectIcon';
+
+export type ProjectItemProps = {
+    label: string;
+    projectName?: string;
+    language?: string;
+    hasIcon?: boolean;
+    href?: string;
+} & Omit<ListItemProps, 'children'>;
+
+export function ProjectItemView({
+                                    label,
+                                    projectName,
+                                    language,
+                                    hasIcon,
+                                    selected,
+                                    className,
+                                    href,
+                                    ...rest
+                                }: ProjectItemProps): ReactElement {
+    return (
+        <ListItem
+            {...rest}
+        >
+            <ListItem.Left>
+                <ProjectIcon
+                    projectName={projectName}
+                    language={language}
+                    hasIcon={hasIcon}
+                />
```

Comment:
I think we have an icon in `ListItem.DefaultContent`. Are there reasons for not using it? Different sizes?